packages/examples/scripts/buildExamples.js

```diff
@@ -18,12 +18,12 @@ async function buildExamples() {
       const exampleFileStat = await fs.stat(exampleFilePath);
 
       if (exampleFileStat.isDirectory()) {
-        const srcPath = pathResolve(exampleFilePath, 'src/index.js');
+        const srcPath = pathResolve(exampleFilePath, 'src');
         const pkgPath = pathResolve(exampleFilePath, 'package.json');
         const pkgStat = await fs.stat(pkgPath);
         const srcStat = await fs.stat(srcPath);
 
-        if (pkgStat.isFile() && srcStat.isFile()) {
+        if (pkgStat.isFile() && srcStat.isDirectory()) {
```

Comment:
I wonder if this will cause issues for us in the future 🤔

Reply:
Why would it? It's just the examples build script, and all it does is check if the `src` folder exists.

Reply:
Will it fail elsewhere if I try to run this script with a `src` folder but no `.js` or `.ts`?

Reply:
Sorry for being unclear.. 😅

Reply:
It will fail if you don't have a `src/index.js` or custom entry point configured, but why would there be an example like that?

Reply:
I think what we have here is good.